packages/project-types/src/webapp/manifest.ts

```diff
@@ -0,0 +1,2960 @@
+/* tslint:disable */
```

Comment:
We have the same file already at https://github.com/SAP/open-ux-tools/blob/main/packages/ui5-config/src/types/manifest.ts.
I fully agree that it should be here, but, please also delete the old one, and make sure that all consumers use your new `project-types` module

Reply:
@tobiasqueck, agree, I'd like to align this with you. There is also an attempt to get this directly from AppDescriptor: https://github.wdf.sap.corp/AppDescriptor/manifest/pull/424

Reply:
Changed it and updated all consumers